maquetado/index.html

```diff
@@ -5,9 +5,21 @@
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
```

Comment:
I think that for organization is better if you import the typography in the `SCSS`